src/Compilers/CSharp/Test/Semantic/Semantics/OutVarTests.cs

```diff
@@ -11162,6 +11688,7 @@ static bool TakeOutParam<T>(T y, out T x)
             var yRef = tree.GetRoot().DescendantNodes().OfType<IdentifierNameSyntax>().Where(id => id.Identifier.ValueText == "y1").Single();
             Assert.Equal("System.Int32", model.GetTypeInfo(yRef).Type.ToTestDisplayString());
 
+            model = compilation.GetSemanticModel(tree);
             var zRef = tree.GetRoot().DescendantNodes().OfType<IdentifierNameSyntax>().Where(id => id.Identifier.ValueText == "z1").Single();
```

Comment:
Changed throughout the file.

---
In reply to: [76287444](https://github.com/dotnet/roslyn/pull/13344#discussion_r76287444) [](ancestors = 76287444)